src/System.IO.Ports/src/System.IO.Ports.csproj

```diff
@@ -33,8 +32,11 @@
     <Compile Include="System\IO\Ports\SerialPinChangedEventArgs.cs" />
     <Compile Include="System\IO\Ports\SerialPinChangedEventHandler.cs" />
     <Compile Include="System\IO\Ports\SerialPort.cs" />
-    <Compile Include="System\IO\Ports\SerialStream.cs" />
     <Compile Include="System\IO\Ports\StopBits.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' != 'netfx' AND '$(TargetsWindows)' == 'true'">
+    <Compile Include="System\IO\Ports\InternalResources.cs" />
+    <Compile Include="System\IO\Ports\SerialStream.cs" />
```

Comment:
Isn't enough common code between the serial stream to use the typical partial class pattern and have SerialStream.Windows.cs and so on?